practice/rain/rain.py

```diff
@@ -0,0 +1,208 @@
+""" Opens a Rain Text File and returns information regarding day with most rain and year with most rain."""
+
+
+from collections import namedtuple
+namedtuple('DayEntry', ['date', 'rainfall'])
+namedtuple('YearEntry', ['year', 'rainfall'])
```

Comment:
These two named tuple calls do nothing. Only the ones that actually assign to names are needed. Also there are two blank lines between imports and code.